File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/FrostmetalDagger.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Dagger;

/* JADX INFO: loaded from: classes3.dex */
public class FrostmetalDagger extends Dagger {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_dagger_frostmetal_dagger_name;
        this.idDescription = R.string.weapon_dagger_frostmetal_dagger_description;
        this.idImage = R.drawable.frostmetal_dagger;
        this.price = 450L;
        this.dexterity = 17;
        this.constitution = 17;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/FrostmetalDagger.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Dagger

class FrostmetalDagger : Dagger() {
    override fun configureProperties() {
        idName = R.string.weapon_dagger_frostmetal_dagger_name
        idDescription = R.string.weapon_dagger_frostmetal_dagger_description
        idImage = R.drawable.frostmetal_dagger
        price = 450L
        dexterity = 17
        constitution = 17
    }
}
